### DISCUSSION TITLE
Is it possible to transform how a value is displayed in table view and frontmatter ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [mdelobelle/metadatamenu#725] [Q&A]
Obsidian| plugin
------- | -------
1.6.7   | v0.8.7

Hi,

For instance, I have a property which type is text/input.
```
Key = repo
Value = https://github.com/mdelobelle/metadatamenu
```

So in frontmatter it’s `repo : https://github.com/mdelobelle/metadatamenu`.

I’d like in the table view, the link still be clickable but displayed as 
[mdelobelle/metadatamenu](https://github.com/mdelobelle/metadatamenu). And when you click to edit, the all `https://github.com/mdelobelle/metadatamenu` be editable. This is my main need.

If in addition, it can be the same in frontmatter until it is edited. Cool.

**Comment by LynetteCullens** [comment]
That's not the proper formatting for an external link within Obsidian.
I do remember there being a link type in the MDM property settings when you add the property to a class, but it doesn't work with incorrect formatting. You will never be able to click a link without formatting it correctly. `[display text](actual link)`

**Reply by LynetteCullens** [reply]
Correction: You will never be able to click a link without formatting it correctly unless you use an additional plugin that overrides the normal settings.
